en/blazor-support.md

```diff
@@ -52,12 +52,38 @@ Generating Blazor application has the following limitations:
 - Bootstrap is turned off in the generated application for now.
 
 Generated components have these limitations:
-- Navigation drawer works only in the overlay state. The mini version does not work too.
-- Date picker works only in dropdown mode.
-- The input group does not support date type.
-- Radio group does not support horizontal mode.
-- Grid cannot generate a toolbar. This disallows the generation of advanced filtering, exporting, pinning, and hiding. Setting cell and row selection do not work simultaneously. Column selection, paging, grouping, and column moving are not supported yet. Filtering is working only in excel style mode.
-- For all components toggle actions are not generated yet.
+- Card:
+    - cannot generate reversed actions.
+    - cannot generate divider in actions column card type.
+- Navigation Bar - cannot generate title.
+- Navigation Drawer - does not support pin state and pin threshold.
+- Avatar:
+    - does not use IgbIcon in generated code.
+    - initials type shows entire contend instead of first two chars.
+- Icon - does not use IgbIcon in generated code.
+- Date Picker:
+    - does not support dialog mode.
+    - does not support months.
+    - does not support action buttons.
+- Floating Action Button - does not use IgbIcon in generated code.
+- Icon Button - does not use IgbIcon in generated code.

```

Comment:
The issue of the Icon button is not the that it doesn't use the icon, but that it's not an actual icon button in Blazor, but a basic button with icon inside. It's the same underlying issue we have with icons that causes this, but still the description can be more accurate